pages/glossary/large-scale-text.md

```diff
@@ -8,7 +8,27 @@ input_aspects:
   - DOM tree
 ---
 
-Text nodes with a [computed](https://www.w3.org/TR/css-cascade-3/#computed-value) [font-size](https://www.w3.org/TR/css-fonts-3/#propdef-font-size) of:
+Text nodes with a [computed](https://www.w3.org/TR/css-cascade-3/#computed-value) [font-size][] of:
 
 - at least 18 [points](https://www.w3.org/TR/css-values/#pt), or
-- at least 14 [points](https://www.w3.org/TR/css-values/#pt) and a [computed](https://www.w3.org/TR/css-cascade-3/#computed-value) [font weight](https://www.w3.org/TR/css-fonts-3/#font-weight-prop) CSS property of 700 or higher.
+- at least 14 [points](https://www.w3.org/TR/css-values/#pt) and a [computed](https://www.w3.org/TR/css-cascade-3/#computed-value) [font weight][] CSS property of 700 or higher.
+
+
+**Exception**:  
+If the text nodes contain Unicode characters in the 4E00–9FFF range ([CJK Unified Ideographs](https://unicode.org/charts/PDF/U4E00.pdf)), these characters are:
+
+- at least 22 [points][], or
+- at least 18 [points][] and a [computed][] [font weight][] CSS property of 700 or higher.
```

Comment:
**Suggestion:** I think we should rephrase the full definition (as it doesn't look like correct English now), in the line of:

> A text node is _large scale text_ if its [computed][] [font-size][] is:
> - at least 18 [points[]; or
> - at least 14 [points][] and its [computed][] [font+weight][] is at least 700.
>
> Exception:
> A text node containing CJK characters is _large scale text_ if …

Maybe even start the definition with "depending whether the text node contains CJK characters, …" 🤔 

Additionally (but not directly related to this PR, so may be taken separately):
* These lists do not match how we write condition lists now; should be "if at least one of the following is true: the [computed][] [font-size][] is at least 18 points; or…"
* Technically, text nodes do not have CSS properties, only their parent element has, so the language used here is a bit incorrect (although not really confusing…)
* Also, not need to say "[font weight][] **CSS propert**" when the name already links to CSS specs.

Reply:
Suggestions applied rewriting the definition and starting with "A text node..."

---

> Additionally (but not directly related to this PR, so may be taken separately):
> 
> These lists do not match how we write condition lists now; should be "if at least one of the following is true: the [computed][] [font-size][] is at least 18 points; or…"

Happy to edit condition lists and include them in the current edits.

> Technically, text nodes do not have CSS properties, only their parent element has, so the language used here is a bit incorrect (although not really confusing…)

You're absolutely right. This was the case in the previous iteration as well. I'm struggling to find a straightforward method to enhance its clarity without adding more complexity to the definition.

> Also, not need to say "[font weight][] CSS propert" when the name already links to CSS specs.

updated.